Orange/preprocess/transformation.py

```diff
@@ -107,7 +107,10 @@ def __init__(self, variable, offset, factor):
         self.factor = factor
 
     def transform(self, c):
-        return (c - self.offset) * self.factor
+        if sp.issparse(c):
+            return c
```

Comment:
You're right, it doesn't 🙈 I was thinking about this and the only think I came up with was to normalise just non-zero values. Though, this is probably also not correct since non-present values are defined as zeros. Normalising everything would break the sparsity structure which is really undesirable for bag of words approaches.

So, should I just normalise non-zero values?